packages/media/src/utils/getHandler.ts

```diff
@@ -1,12 +1,30 @@
 import { detectDevice } from "mediasoup-client";
+import { BuiltinHandlerName } from "mediasoup-client/lib/types";
+import { UAParser } from "ua-parser-js";
```

Comment:
Can we use webrtc-adapter as we do in other modules? I believe we get the browser name and major version there as well.